update transitions-enterleave.md (#1353)
vuejs/docs: src/guide/transitions-enterleave.md

With one attribute addition, we've fixed that original transition without having to add any special styling.

> We can use this to coordinate more expressive movement, such as a folding card, as demonstrated below. Try hovering over each image and clicking the replace button.

Author: ```suggestion
We can use this to coordinate more expressive movement, such as a folding card, as demonstrated below. Try hovering over each image and clicking the replace button:
```

With one attribute addition, we've fixed that original transition without having to add any special styling.

> We can use this to coordinate more expressive movement, such as a folding card, as demonstrated below (try clicking the replace button). It's actually two elements transitioning between each other, but since the beginning and end states are scaling the same: horizontally to 0, it appears like one fluid movement. This type of sleight-of-hand can be very useful for realistic UI microinteractions:

Reviewer: ```suggestion
We can use this to coordinate more expressive movement, such as a folding card, as demonstrated below. Try hovering over each image and clicking the replace button.
```

> <common-codepen-snippet title="Transition Modes Flip Cards" slug="76e344bf057bd58b5936bba260b787a8" :editable="false" />

Reviewer: ```suggestion
<common-codepen-snippet title="Transition Modes Flip Cards" slug="76e344bf057bd58b5936bba260b787a8" :editable="false" />

It's actually two elements transitioning between each other, but since the beginning and end states are scaling the same: horizontally to 0, it appears like one fluid movement. This type of sleight-of-hand can be very useful for realistic UI micro-interactions!

```
